awsutil/awsutil.go

```diff
@@ -57,5 +57,8 @@ func TimeDifference(server string) (time.Duration, error) {
 	}
 
 	f, err := strconv.ParseFloat(string(submatched[1]), 64)
+	if err != nil {
+		return time.Duration(0), err
```

Comment:
I can see you took this style from code above, but FWIW, because of how constants in Go work, simply `return 0, err` would also work.